### PR TITLE
Update readme to add asset update script and fixes

### PR DESCRIPTION
**Pull request by dcmiddle**
Add instructions for shutdown, running sample updates, running after the
first install.

Remove/reword sawtooth-core references.

Signed-off-by: Dan Middleton <dan.middleton@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,22 +45,34 @@ Setup Instructions:
...
+2. $ docker-compose down
+
+Subsequent Runs:
+1. $ docker-compose up
+2. Open a new terminal
+3. Navigate to sawtooth-supply-chain/server
+4. $ npm run make-asset
+5. $ npm run seed-sample-assets
+6. Navigate your browser to localhost:3000/asset
+7. $ npm run update-sample-assets
```
Comment (peterschwarz) [inline README.md:62]: I would suggest having this be step 6

**Comment by dcmiddle** [comment]
I think its good to verify first that you have the basic setup running and then throw the updates at it.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 ![Hyperledger Sawtooth](images/sawtooth_logo_light_blue-small.png)  
 
-Hyperledger Sawtooth
--------------
+Sawtooth Supply Chain
+-----------------
+
+This is a distributed application to help you trace the provenance and other contextual information of any asset.
+ It can be used as-is or customized for different usages.
+This supply chain dApp runs on top of Hyperledger Sawtooth, an enterprise blockchain.
```
Comment (delventhalz) [inline README.md:9]: > dApp

File: README.md (hunk 1 above)
Comment (delventhalz) [reply README.md:62]: If you wanted to view the assets updating live, you would want to navigate there first

**Review by delventhalz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dcmiddle) [reply README.md:9]: Not sure what your feedback is. Do you not like the term dApp? I know I don't, but there it is anyway.  :)

File: README.md (hunk 1 above)
Comment (dcmiddle) [reply README.md:62]: I'm going to leave the order as-is (go to browser first, then launch live updates), unless someone has a strong objection?

**Review by delventhalz — APPROVED** [review]

**Pull request merged**